src/molecules/BottomSheet/BottomSheetSectionList.native.js

```diff
@@ -0,0 +1,10 @@
+import { SectionList } from 'react-native';
+
+const BottomSheetSectionList = () => null;
+
+BottomSheetSectionList.displayName = 'BottomSheetSectionList';
+
+/**Not sure if this works :( */
+BottomSheetSectionList.propTypes = SectionList;
```

Comment:
I dont think it does anything. Not sure what to do here either

Reply:
why do you need this in first place?

Reply:
Yes, right, it does nothing. We can remove this.

Reply:
Needed it to pass RN SectionList propTypes to our BottomSheetSectionList.